experimenter/tests/integration/nimbus/android/test_fenix_integration.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture
+def experiment_slug():
+    return "firefox-fenix-test-experiment"
```

Comment:
Why not `EXPERIMENT_SLUG = "firefox-fenix-test-experiment"`? See my other comment about fixture magic.

Reply:
This is used in another fixture. I was trying to keep the methods and calls similar to how we do it on desktop. No magic here, just returning a string 😃